extensions/data-transfer/portability-data-transfer-microsoft/src/main/java/org/datatransferproject/transfer/microsoft/photos/MicrosoftPhotosImporter.java

```diff
@@ -60,6 +62,8 @@
  */
 public class MicrosoftPhotosImporter implements Importer<TokensAndUrlAuthData, PhotosContainerResource> {
 
+  private static final List<String> FORBIDDEN_CHARACTERS = Arrays.asList("~\"#%&*:<>?/\\{|}.");
```

Comment:
I'm a little confused why this has to be a list of strings instead of a list of chars - can you change it to either be 1 string (and get rid of the list) or a list of chars? 


Also can you add a comment pointing to the docs?

Reply:
Changed to string, i meant to make it a list of chars initially, but string is easier to read i feel.

Added the doc in the MicrosoftTransmogrificationConfig